### DISCUSSION TITLE
多次运行导出Excel有概率报错，重新运行又好了

### DISCUSSION THREAD

**Discussion by zrcshmily** [qax-os/excelize#1526] [Q&A]
goroutine 213 [running]:
runtime.throw(0x5a1578, 0x21)
        d:/go/src/runtime/panic.go:774 +0x79 fp=0xc00254b678 sp=0xc00254b648 pc=0x42ecd9
runtime.mapaccess1_faststr(0x561320, 0xc003800120, 0xc005ac66c0, 0x18, 0xc002bb2140)
        d:/go/src/runtime/map_faststr.go:21 +0x456 fp=0xc00254b6e8 sp=0xc00254b678 pc=0x411d06
github.com/360EntSecGroup-Skylar/excelize.(*File).workSheetReader(0xc0029c6070, 0xc00095ebd0, 0x25, 0x780000002e)
        C:/Users/Administrator/go/pkg/mod/github.com/360!ent!sec!group-!skylar/excelize@v1.4.1/excelize.go:107 +0x1ee fp=0xc00254b758 sp=0xc00254b6e8 pc=0x5121fe
github.com/360EntSecGroup-Skylar/excelize.(*File).GetRows(0xc0029c6070, 0xc00095ebd0, 0x25, 0xa, 0x599d06, 0xf)
        C:/Users/Administrator/go/pkg/mod/github.com/360!ent!sec!group-!skylar/excelize@v1.4.1/rows.go:33 +0x64 fp=0xc00254bda8 sp=0xc00254b758 pc=0x513e34

-----------------------------------------------------------------------
ompress/flate.(*decompressor).huffSym(0xc001abe000, 0xc001abe850, 0x106, 0x0, 0x0)
        d:/go/src/compress/flate/inflate.go:707 +0x25b
compress/flate.(*decompressor).huffmanBlock(0xc001abe000)
        d:/go/src/compress/flate/inflate.go:564 +0x53d
compress/flate.(*decompressor).nextBlock(0xc001abe000)
        d:/go/src/compress/flate/inflate.go:327 +0x17d
compress/flate.(*decompressor).Read(0xc001abe000, 0xc000e38c00, 0x200, 0x200, 0x0, 0xc00400d9f0, 0x4cb010)
        d:/go/src/compress/flate/inflate.go:347 +0x7e
archive/zip.(*pooledFlateReader).Read(0xc00191b1e0, 0xc000e38c00, 0x200, 0x200, 0x0, 0x0, 0x0)
        d:/go/src/archive/zip/register.go:90 +0xd2
archive/zip.(*checksumReader).Read(0xc005ad5db0, 0xc000e38c00, 0x200, 0x200, 0x30, 0x28, 0xc00400dab0)
        d:/go/src/archive/zip/reader.go:198 +0x73
bytes.(*Buffer).ReadFrom(0xc00334f800, 0x11a0030, 0xc005ad5db0, 0x11a0070, 0xc00334f800, 0x1101)
        d:/go/src/bytes/buffer.go:204 +0xbb
io.copyBuffer(0x5c8ac0, 0xc00334f800, 0x11a0030, 0xc005ad5db0, 0x0, 0x0, 0x0, 0x1b8, 0x8900000000000000, 0x0)
        d:/go/src/io/io.go:388 +0x2f4
io.Copy(...)
        d:/go/src/io/io.go:364
github.com/360EntSecGroup-Skylar/excelize.readFile(0xc0016b1340, 0x59c6a1, 0x13, 0x0)
        C:/Users/Administrator/go/pkg/mod/github.com/360!ent!sec!group-!skylar/excelize@v1.4.1/lib.go:63 +0x112
github.com/360EntSecGroup-Skylar/excelize.ReadZipReader(0xc0048379c0, 0xc00334e900, 0x1c524, 0xc0048379c0, 0x0)
        C:/Users/Administrator/go/pkg/mod/github.com/360!ent!sec!group-!skylar/excelize@v1.4.1/lib.go:29 +0x96
github.com/360EntSecGroup-Skylar/excelize.OpenReader(0x5c8d20, 0xc000e2e098, 0x0, 0x30000c000000000, 0xc000e2e098)
        C:/Users/Administrator/go/pkg/mod/github.com/360!ent!sec!group-!skylar/excelize@v1.4.1/excelize.go:68 +0x115
github.com/360EntSecGroup-Skylar/excelize.OpenFile(0xc000124400, 0x13, 0x0, 0x0, 0x0)
        C:/Users/Administrator/go/pkg/mod/github.com/360!ent!sec!group-!skylar/excelize@v1.4.1/excelize.go:48 +0xe9
yxlserver/cmd/exceltool/excel.(*Excel).ExcelGetCfgStructByFile(0xc0000b2080, 0xc000124400, 0x13)

**Comment by xuri** [comment]
Thanks for your feedback. Could you show us a complete, standalone example program or reproducible demo? Please also try to upgrade the lasted version.

**Reply by zrcshmily** [reply]
thanks，up version to v2 is ok
